frontend/app/mocks/db.ts

```diff
@@ -170,30 +170,30 @@ db.subscription.create({
 
 db.subscriptionConfirmationCode.create({
   id: '0000001',
-  email: 'unverified@example.com',
-  confirmationCode: '0001',
+  userId: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
```

Comment:
```suggestion
  userId: unverifiedUser.id,
```

(Same comment applies for all occurences where `userId: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6'` is used below)

Reply:
fixed